suite-native/module-send-receive/src/screens/ReceiveScreen.tsx

```diff
@@ -27,6 +28,14 @@ export const ReceiveScreen = ({
 
     if (!account) return <ErrorMessage errorMessage={`Account ${accountKey} not found.`} />;
 
+    const handleShowAddress = () => {
```

Comment:
Some function have analytics suffix and this one doesn't. Please unify it across the whole PR or it's going to be a big mess. I suggest we drop the analytics suffix everywhere. It's not necessary I think